pkg/ddc/juicefs/type.go

```diff
@@ -54,6 +54,7 @@ type Configs struct {
 }
 
 type Worker struct {
+	NotPrivileged   bool                   `yaml:"notPrivileged,omitempty"`
```

Comment:
This supposed to be `unprivileged`?

Reply:
How about using `privileged` ? unPrivileged is not straight forward.